docker-entrypoint.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+
+DATABASE_MIGRATE=""
+HOST="0.0.0.0"
+LOGLEVEL="warning"
+PORT="8000"
+
+while getopts "dh:l:p:" opt; do
+    case $opt in
+        d)
+            DATABASE_MIGRATE="True"
```

Comment:
So if i understand correctly if you put different arguments in your Dockerfile as arguments you would overwrite these values? And this is handy because we want to execute alembic right?

Reply:
yes. but sometimes you might not want to execute it. maybe because your CI/CD manages the migration (this is often the case. that why i made it optional